abi/cbor_bytes.go

```diff
@@ -0,0 +1,51 @@
+package abi
+
+import (
+	"fmt"
+	"io"
+
+	cbg "github.com/whyrusleeping/cbor-gen"
+	"golang.org/x/xerrors"
+)
+
+type CborBytes []byte
```

Comment:
We should consider putting this into cbor-gen (and using cborgen's ReadByteArray/WriteByteArray.

Reply:
...future...improvement...

Reply:
https://github.com/whyrusleeping/cbor-gen/issues/75